lib/clinic_finder.rb

```diff
@@ -9,5 +9,25 @@ class ClinicFinder
   def initialize(yml_file)
 		@clinics = ::YAML.load_file(yml_file)
   end
+
+  def locate_cheapest_clinic(gestational_age:, naf_clinics_only: false)
+    @helper = ::ClinicFinder::GestationHelper.new(gestational_age)
+    @gestational_tier = @helper.gestational_tier
+    decorate_data(available_clinics)
+  end
+
+  private def decorate_data(data)
+    sorted_clinics = []
+    three_cheapest(data).map { |clinic_array| sorted_clinics << { clinic_array.first => clinic_array.last } }
```

Comment:
can you add a comment about what's going on here? I think I get it but having explicit would rule